README.md

```diff
@@ -1555,6 +1555,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Rupesh Piwal](https://github.com/Rupesh-Piwal)
   - [Russ Perry](https://github.com/rperry99)
   - [Rutam Bhagat](https://github.com/RutamBhagat)
+  - [Ruchi Sharma](https://github.com/ruchidh)
```

Comment:
Please add your name in line 1551, after Ruben Cliford Gomes.
Let us know if you need any help with that. 🙂